src/opm/input/eclipse/Schedule/Network/ExtNetwork.cpp

```diff
@@ -113,15 +146,18 @@ std::optional<Branch> ExtNetwork::uptree_branch(const std::string& node) const {
     if (branches.size() == 1)
         return std::move(branches[0]);
 
-    throw std::logic_error("Bug - more than upstree branch for node: " + node);
+    throw std::logic_error("Bug - more than one uptree branch for node: " + node);
 }
 
 
 std::vector<Branch> ExtNetwork::downtree_branches(const std::string& node) const {
-    if (!this->has_node(node))
-        throw std::out_of_range("No such node: " + node);
-
     std::vector<Branch> branches;
+
+    if (!this->has_node(node)) {
```

Comment:
This should be at the beginning of the function, before `std::vector<Branch> branches;`.